### DISCUSSION TITLE
Stack underflow issues.

### DISCUSSION THREAD

**Discussion by Seraphim-AT** [airsdk/Adobe-Runtime-Support#2058] [General]
Hey there! 

I have a weird issues I have not seen for a long time, they appeared in latest 33.1.1.889 and 33.1.1.929 releases, the same issues in both. so, in simple piece of code, like:

```
var count: int = /* any value */;
while (--count >= 0) {
    /* any code here */
}
```

I'm getting a compiler alert about an error:

`Error:(581, 0) [...]: Generated byte code contains an operand stack underflow.`

it's just an example, I have much more similar errors in very typical code, which works fine in 33.1.1.856 version and above.

**Comment by ajwfrost** [comment]
Hi

Can I please check, are you just compiling via mxmlc to get this? or is this happening when packaging (ADT) - and if so, which target output? Are you able to confirm what platform you're using (Windows/Mac/etc)?

We've not been able to reproduce this using mxmlc or when packaging up an IPA. Not sure if it's only happening when certain conditions are being hit though, so if you are able to provide the full .as file and maybe the command-line, we can double-check it...

thanks
